diff --git a/HdmiCecSink/HdmiCecSink.cpp b/HdmiCecSink/HdmiCecSink.cpp
index a45ebfa..f95c3ea 100644
--- a/HdmiCecSink/HdmiCecSink.cpp
+++ b/HdmiCecSink/HdmiCecSink.cpp
@@ -26,7 +26,7 @@
 
 
 #define TR181_HDMICECSINK_CEC_VERSION "Device.DeviceInfo.X_RDKCENTRAL-COM_RFC.Feature.HdmiCecSink.CECVersion"
-
+extern "C" void __gcov_exit();
 
 #define API_VERSION_NUMBER_MAJOR 1
 #define API_VERSION_NUMBER_MINOR 3
@@ -147,6 +147,7 @@ namespace WPEFramework
            _service->Unregister(&_notification);
            _service->Release();
            _service = nullptr;
+           __gcov_exit();
            LOGINFO("HdmiCecSink plugin is deactivated. Successfully deactivated HdmiCecSink Plugin");
         }
 
diff --git a/HdmiCecSink/HdmiCecSinkImplementation.cpp b/HdmiCecSink/HdmiCecSinkImplementation.cpp
index f4260da..a6c55bf 100644
--- a/HdmiCecSink/HdmiCecSinkImplementation.cpp
+++ b/HdmiCecSink/HdmiCecSinkImplementation.cpp
@@ -107,7 +107,7 @@ static std::vector<DeviceFeatures> deviceFeatures = {DEVICE_FEATURES_TV};
 #define API_VERSION_NUMBER_MAJOR 1
 #define API_VERSION_NUMBER_MINOR 3
 #define API_VERSION_NUMBER_PATCH 7
-
+extern "C" void __gcov_exit();
 using PowerState = WPEFramework::Exchange::IPowerManager::PowerState;
 
 namespace WPEFramework
@@ -704,6 +704,7 @@ namespace WPEFramework
                 LOGINFO("HdmiCecSink plugin device::Manager::DeInitialize failed");
                 LOG_DEVICE_EXCEPTION0();
              }
+             __gcov_exit();
        }
 
        Core::hresult HdmiCecSinkImplementation::Configure(PluginHost::IShell *service)
